main.ts

```diff
@@ -9,15 +9,36 @@
 /// <reference lib="deno.unstable" />
 
 import { ServerContext } from "$fresh/server.ts";
+import { Fragment, h } from "$fresh/runtime.ts";
 import { ConnInfo, serve } from "$std/http/server.ts";
 import { router } from "$router";
 import { createReporter, Reporter } from "$ga";
 import { accepts } from "$oak_commons";
+import { setup } from "$doc_components/services.ts";
 
 import manifest from "./fresh.gen.ts";
 
 import { routes as completionsV2Routes } from "./completions_v2.ts";
 
+const docland = "https://doc.deno.land/";
+await setup({
+  resolveHref(current, symbol) {
+    return symbol
+      ? `${docland}https://deno.land${current}/~/${symbol}`
```

Comment:
I think this is wrong.. When I click a link like "assertsEquals" in https://dotland-mq4cvd8dncj0.deno.dev/std@0.143.0/testing/asserts.ts

[assertEquals](https://doc.deno.land/https://deno.landhttps://dotland-mq4cvd8dncj0.deno.dev/std@0.143.0/testing/asserts.ts/~/assertEquals)